src/main/java/com/oneops/proxy/web/ErrorController.java

```diff
@@ -52,8 +53,27 @@ public ErrorController(ErrorAttributes errAttrs, ServerProperties serverProperti
   public @ResponseBody ResponseEntity<ErrorResponse> errorResponse(
       HttpServletRequest req, HttpServletResponse res) {
     Map<String, Object> body = getErrorAttributes(req, isIncludeStackTrace(req, MediaType.ALL));
-    HttpStatus status = getStatus(req);
 
+    /*---------------Update status code and format error respose for RequestRejectedException------------------*/
+    Object message = null;
+    Object path = null;
+    String exceptionName = body.get("exception").toString();
+    final String REQUEST_REJECTED_EXCEPTION = "RequestRejectedException";
+
+    if (exceptionName.contains(REQUEST_REJECTED_EXCEPTION)) {
+      try{
+        message = URLDecoder.decode(body.get("message").toString(), "UTF-8");
+        path = URLDecoder.decode(body.get("path").toString(), "UTF-8");
+      }catch(UnsupportedEncodingException ue){}
+      message = message.toString().replace(exceptionName +":", "");
```

Comment:
This is a possible NPE. What  happens if `message = URLDecoder.decode(body.get("message").toString(), "UTF-8");` throws exception? `message.toString()` will throw NPE..rt?